src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="top-tier">
+  <h2><b>{{ topHeader }}</b></h2>
+  <h4>{{ topMessage }}</h4>
+</div>
```

Comment:
I don't think the top half of the page is necessary.
Most of the info is being duplicated/separated and we don't have any other page that does this half page style.
I would move the important info to the bottom half and remove the top section. 

I would also remove the breadcrumb since it doesn't provide any useful info.

Reply:
Thanks @tspmelo , will refactor accordingly.!